app/views/validation/_message.html.erb

```diff
@@ -1,11 +1,11 @@
 <p><%= t(message.category) %>: <strong><%= t(message.type) %>
 <% if message.row %>
-  on row <%= message.row %>  
+  on row <%= message.row %>
 <% end %>
 <% if message.column %>
   in column <%= message.column %>
 <% end %></strong></p>
-<% if message.content %>
+<% if message.content != ""%>
```

Comment:
Better still, you could do `unless message.conent.blank?`, which would cover empty strings and nil values. Probably not strictly necessary, but belt and braces :smile: